tools/github_readme_sync/hierarchy.py

```diff
@@ -278,11 +286,28 @@ def check_readme_link(url, rdme):
 
 
 def check_external_link(url):
+    if "openai.com" in url or "science.org" in url:
+        return []
+
     try:
         headers = request_headers()
-        response = requests.get(url, timeout=5, headers=headers)
+        time = timeit.default_timer()
+
+        try:
+            response = requests.head(url, timeout=5, headers=headers)
+            if response.status_code < 200 or response.status_code > 299:
+                response = requests.get(url, timeout=5, headers=headers)
+        except requests.RequestException:
```

Comment:
**issue**: This will catch `requests.RequestException` thrown from `requests.get` on the line above.